codegen/facelift/templates/Struct.template.h

```diff
@@ -44,7 +44,9 @@
 
 // Dependencies
 {% for field in struct.fields %}
```

Comment:
Did you try the following ?
```suggestion
{% for field in struct.fields -%}
{{field.type.requiredInclude}}
{-% endfor %}
```

If that works, I would prefer that approach.

Reply:
Good idea, but unfortunately it doesn't works, there is such code generated:

 #include "facelift/example/addressbook/SubInterface.h"#include "facelift/example/addressbook/Contact.h"#include "facelift/example/addressbook/FailureReason.h"